integreat_cms/cms/templates/contacts/contact_list.html

```diff
@@ -89,6 +89,20 @@ <h1 class="heading">
                 <option>
                     {% translate "Select bulk action" %}
                 </option>
+                {% if is_archive %}
+                    <option data-bulk-action="{% url "bulk_restore_contacts" region_slug=request.region.slug %}">
+                        {% translate "Restore contacts" %}
+                    </option>
+                {% else %}
+                    <option data-bulk-action="{% url "bulk_archive_contacts" region_slug=request.region.slug %}">
+                        {% translate "Archive contacts" %}
+                    </option>
+                {% endif %}
```

Comment:
```suggestion
    {% if perms.cms.change_contact %}
                {% if is_archive %}
                    <option data-bulk-action="{% url "bulk_restore_contacts" region_slug=request.region.slug %}">
                        {% translate "Restore contacts" %}
                    </option>
                {% else %}
                    <option data-bulk-action="{% url "bulk_archive_contacts" region_slug=request.region.slug %}">
                        {% translate "Archive contacts" %}
                    </option>
                {% endif %}
            {% endif %}
```

These bulk actions should only be available to users who have the permission to modify contacts

Reply:
The check is already there in Line 86